lib/PlaidClient.js

```diff
@@ -332,4 +332,16 @@ Client.prototype.getCategories =
 Client.prototype.resetLogin =
   requestWithAccessToken('/sandbox/item/reset_login');
 
+// sandboxPublicTokenCreate(String, Array, Object?, Function) - sandbox only
+Client.prototype.sandboxPublicTokenCreate =
+function(institution_id, initial_products, options, cb) {
+  return this._authenticatedRequest({
+    path: '/sandbox/public_token/create',
+    body: {
+      institution_id: institution_id,
+      initial_products: initial_products,
+    },
+  }, options, cb, true);
```

Comment:
are there actually`options` for this route?

I'm realizing the fact that `_authenticatedRequest` [injects options onto the request](https://github.com/plaid/plaid-node/blob/master/lib/PlaidClient.js#L48) in this way is somewhat confusing...

pretty sure I devised this scheme to support the variadic arguments behavior when you use the callback style... I wonder if anyone is benefitting from that...

Reply:
👍